tests/start-cypress-tests.sh

```diff
@@ -30,6 +30,22 @@ fi
 echo "Logging into Kube API server..."
 oc login --server=$CYPRESS_OC_CLUSTER_URL -u $CYPRESS_OC_CLUSTER_USER -p $CYPRESS_OC_CLUSTER_PASS --insecure-skip-tls-verify
 
+echo "Checking RedisGraph deployment."
+oc get pod -n open-cluster-management | grep search-redisgraph-0 | grep Running
+if [ "$?" == "1" ]; then
+  echo "RedisGraph not found, deploying and waiting 60 seconds for the search-redisgraph-0 pod."
+  oc set env deploy search-operator DEPLOY_REDISGRAPH="true" -n open-cluster-management
+
+  # I'd prefer to do this from the operator but here is easier.
```

Comment:
Was intended as a TODO for search repo. We plan to move to search-operator later to deploy all pods, but currently use a combination of chart and operator.

Reply:
Yes, I hope to complete this TODO before we merge. This is also the reason why the code inside Cypress doesn't do the restart because ideally the search operator will do it.

Reply:
Moved to operator in [PR](https://github.com/open-cluster-management/search-operator/pull/57)